gtsfm/utils/metrics.py

```diff
@@ -141,15 +141,15 @@ def compute_translation_angle_metrics(
     for (i1, i2) in i2Ui1_dict:
         i2Ui1 = i2Ui1_dict[(i1, i2)]
         angles.append(comp_utils.compute_translation_to_direction_angle(i2Ui1, wTi_list[i2], wTi_list[i1]))
-    return compute_errors_statistics(angles)
+    return GtsfmMetric("translation_to_direction_angle_deg", np.array(angles))
```

Comment:
is "translation_angle_deg" better than "translation_to_direction_angle_deg"?

Reply:
Ive changed it, but Im not sure if everyone understands what translation angle is.